src/docs/command-line.dox

```diff
@@ -187,7 +187,8 @@ format.
 
 @par <code>-p</code>
 Dump in human-readable format (pretty-print). The \c -p flag is incompatible
-with the \c load command.
+with the \c load command. The \c -p flag can be combined with \c -x. In this case raw data elements
+will be formatted like \c -x does that.
```

Comment:
Please fix all occurrences of `like -x does that` to something ilke "...raw data elements will be formatted like \c -x with hexadecimal encoding". The `does that` is not clear to the reader.